website/docs/r/container_cluster.html.markdown

```diff
@@ -172,6 +172,8 @@ output "cluster_ca_certificate" {
 
 * `remove_default_node_pool` - (Optional) If true, deletes the default node pool upon cluster creation.
 
+* `resource_labels` - (Optional) The GCE resource labels (key/value pairs) to be applied to the cluster.
```

Comment:
Could we specify this is a map, just so users know how to supply the information?

Reply:
I couldn't find an example of this in the docs (where labels are set elsewhere), but have mentioned its a map

Reply:
Gah, sorry to not have a good example for you. Thank you for addressing it anyways!